File: check_n8n_db.js
const sqlite3 = require('C:/Users/usuario/AppData/Roaming/npm/node_modules/sqlite3').verbose() || require('sqlite3').verbose();
const db = new sqlite3.Database('C:/Users/usuario/.n8n/database.sqlite');

db.all("SELECT id, name, active FROM workflow_entity WHERE active = 1", (err, rows) => {
    if (err) { console.error(err); return; }
    console.log("--- FLUJOS ACTIVOS EN BD ---");
    console.log(JSON.stringify(rows, null, 2));
    
    db.all("SELECT * FROM webhook_entity", (err, webhooks) => {
        if (err) { console.error(err); return; }
        console.log("--- WEBHOOKS REGISTRADOS ---");
        console.log(JSON.stringify(webhooks, null, 2));
        db.close();
    });
});

File: scratch/check_n8n_db.py
import sqlite3
import json
import os

db_path = r'C:\Users\usuario\.n8n\database.sqlite'

if not os.path.exists(db_path):
    print(f"ERROR: No se encuentra la base de datos en {db_path}")
    exit(1)

conn = sqlite3.connect(db_path)
cursor = conn.cursor()

print("--- LISTADO DE FLUJOS EN BD ---")
cursor.execute("SELECT id, name, active FROM workflow_entity")
rows = cursor.fetchall()
for row in rows:
    print(f"ID: {row[0]} | Nombre: {row[1]} | Activo: {row[2]}")

conn.close()
